.github/workflows/rust-ci-cd.yml

```
@@ -563,78 +344,152 @@ jobs:
+      - test
+      - test-prowo-interaction
+      - test-prowo-edge-cases
+      - test-performance
+      - test-prowo-docker
     runs-on: blacksmith-2vcpu-ubuntu-2404
     if: always()
-
     steps:
       - name: 📈 Create test summary
+        if: needs.allow-run.outputs.RUN_CI_CD == 'true' == 'true'
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)